src/components/ProductHero/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { LandingPageCallToAction } from '../LandingPage/LandingPageCallToAction'
+
+import HeroAnimation720x1300 from './svgs/hero-720x1300.svg'
```

Comment:
I think importing this injects all of these SVGs into the JS of every page load, so you notice this page is super slow. You should be able to direct link the svg instead by doing `<img src={HeroAnimation720x1300} />`.


You might also be able to  use [webm](https://evilmartians.com/chronicles/images-done-right-web-graphics-good-to-the-last-byte-optimization-techniques#animations-on-the-web), as it could be smaller, but maybe try the above approach first.

Reply:
Ah, I didn't think animated svgs could be loaded as <img> tags - will give it a shot.

As it's an animated SVG and not a video, I'm not sure the webm format would apply? Though I'm not fully up to date on this stuff, so I could be wrong.

Reply:
You could convert it into a video, as SVG uses a lot of the users' computing power to calculate the animation, whereas a video is much easier to display (which will result in a faster feeling website).